### ISSUE TITLE
Deposit form remove button accessibility

### ISSUE THREAD

**Issue by justinlittman** [sul-dlss/happy-heron#3181]
There are multiple "Remove" buttons that perform different actions. Form field labels should be unique.

**Comment by justinlittman** [comment]
@astridu What is meant by "unique"? I get that the remove button for a keyword should be labeled differently than the remove button for an author. But does each author remove button need to be labeled uniquely? And if so, what should each label be (given that the user may not have yet populated the author info or may change existing author info)?

(I tried consulting various accessibility references but couldn't find a clarification.)

**Comment by astridu** [comment]
@justinlittman Is this for the edit deposit form? I think what they mean is that they may have the screenreader go through the links, and if the links all say 'remove' or 'remove author', the user may not know which one they are removing. We got around this in Monarch by putting the more descriptive label in an ARIA label associated with the button. So we could leave the button as 'remove', and then the ARIA label said 'remove [specific item]' Is there a way to put the name of the author in the ARIA label? I'm assuming if there is a 'remove' button, there is an author listed.

**Comment by justinlittman** [comment]
We can make the label "remove [specific item]", e.g., "remove Justin Littman". However, (1) when you select "Add another contributor", there is no "specific item" (and there can be multiple new contributors) and (2) a user can change the author, such that the label is incorrect.

**Comment by astridu** [comment]
Good points. This would be good to raise at the training happening at 1pm.

**Comment by astridu** [comment]
@justinlittman Spoke to Kevin. He said our approach is ok because it is equivalent to what would happen to any other user. So for the 2 use cases you pointed out:

1. User changes "Jane Smith" to "June Smith" but hasn't saved yet. User hits 'Remove bu

**Comment by justinlittman** [comment]
One additional clarification question: A user can add multiple blank authors. Should each have the label "Remove blank author"? (Since this isn't unique when there are multiple.)

**Comment by astridu** [comment]
Yes.
